fboss/qsfp_service/test/hw_test/HwTransceiverUtils.cpp

```diff
@@ -692,6 +740,9 @@ void HwTransceiverUtils::verifyDiagsCapability(
              *mediaIntfCode == MediaInterfaceCode::LR4_2x400G_10KM ||
              *mediaIntfCode == MediaInterfaceCode::DR4_2x400G ||
              *mediaIntfCode == MediaInterfaceCode::DR4_2x800G ||
+             *mediaIntfCode == MediaInterfaceCode::DR2_4x400G ||
+             *mediaIntfCode == MediaInterfaceCode::DR1_8x200G ||
+             *mediaIntfCode == MediaInterfaceCode::DR1_8x100G ||
```

Comment:
Not required, as this switch case is on moduleMediaInterface which is always DR4_2x800G